docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -618,24 +618,26 @@ link:https://tinkerpop.apache.org/docs/x.y.z/reference/#asDate-step[reference]
 [[asString-step]]
 === asString()
 
-*Description:* Returns the value of incoming traverser as strings
+*Description:* Returns the value of incoming traverser as strings, or if `Scope.local` is specified, returns each element inside
+incoming list traverser as string.
 
-*Syntax:* `asString()`
+*Syntax:* `asString()` | `asString(local)`
 
 [width="100%",options="header"]
 |=========================================================
 |Start Step |Mid Step |Modulated |Domain |Range
-|N |Y |N |`any` |`String`
+|N |Y |N |`any` |`String`/`List`
 |=========================================================
 
 *Arguments:*
 
-None
+* `scope` - Determines the type of travers it operates on. Both scope will operate on the level of individual traversers.
+The `global` scope will operate on individual traverser. The `local` scope will operate on list traverser.
 
 Null values from the incoming traverser are not processed and remain as null when returned.
 
 *Exceptions*
-* If the incoming traverser is a non-String value then an `IllegalArgumentException` will be thrown.
+None
```

Comment:
Maybe add some explanation about what happens for non-String traversers for Global Scope and non-list traversers for local scope.

Reply:
I believe that is explained above under arguments